src/libaktualizr/package_manager/packagemanagerinterface.cc

```diff
@@ -96,18 +96,19 @@ bool PackageManagerInterface::fetchTarget(const Uptane::Target& target, Uptane::
       LOG_WARNING << "Skipping download of target with length 0";
       return true;
     }
-    DownloadMetaStruct ds(target, std::move(progress_cb), token);
+    std::unique_ptr<DownloadMetaStruct> ds{new DownloadMetaStruct(target, std::move(progress_cb), token)};
```

Comment:
`make_unique`?

Reply:
Yeah, it makes sense to make it unique, however, I have to use the custom make_unique (std_::make_unique) since we don't support c++14 by default.